### PR TITLE
Add to table mouseover and header height prop

### PR DESCRIPTION
**Pull request by FlaviooLima**
| - | - |
| --- | --- |
| Issue | We need `mouseover` and a prop to set the height of the table header to be able to build the dynamic component |
| Decisions |  Pass those trough props and apply in the code |
| Animated GIF |   |

### CODE REVIEW

File: vue/components/ui/molecules/table/table.vue [hunk 1]
```diff
@@ -244,6 +248,10 @@ export const Table = {
         reverse: {
             type: Boolean,
             default: false
+        },
+        headerHeight: {
+            type: Number,
+            default: 36
```
Comment (NFSS10) [inline vue/components/ui/molecules/table/table.vue]: make it optional: `default: null`.

File: vue/components/ui/molecules/table/table.vue [hunk 2]
```diff
@@ -275,6 +283,9 @@ export const Table = {
             const order = this.reverseData ? "ascending" : "descending";
             return this.sortData === column ? `active ${order}` : "";
         },
+        columnStyle(column) {
+            return { width: column.width, height: `${this.headerHeight}px` };
```
Comment (NFSS10) [inline vue/components/ui/molecules/table/table.vue]: We are using px for sizes. Also, make it optional: 
`            const base = {};
            if (this.width) base.width = `${this.width}px`;
            if (this.headerHeight) base.height = `${this.headerHeight}px`;
            return base;`

File: vue/components/ui/molecules/table/table.vue [hunk 3]
```diff
@@ -283,6 +294,9 @@ export const Table = {
         },
         onClick(item, index) {
             this.$emit("click", item, index);
+        },
+        mouseOver(item, index) {
```
Comment (NFSS10) [inline vue/components/ui/molecules/table/table.vue]: prefix events with on: mouseOver -> onMouseOver

**Comment by FlaviooLima** [comment]
No need for this any more

**Pull request closed without merging**